views/default/css/admin.php

```diff
@@ -1300,42 +1300,43 @@
 /* ***************************************
 	ICONS
 *************************************** */
+.fa {
```

Comment:
IMHO, can use .elgg-icon and drop the font-size (it should inherit the font-size of the text)